cdi/src/test/java/org/hibernate/validator/test/internal/cdi/injection/InjectionOfCustomProviderTest.java

```diff
@@ -78,43 +74,43 @@ public static JavaArchive createDeployment() {
 
 	@Test
 	public void testInjectionOfDefaultFactory() throws Exception {
-		assertNotNull( defaultValidatorFactory );
-		assertTrue( defaultValidatorFactory instanceof MyValidatorFactory );
-		assertTrue( defaultValidatorFactory.unwrap( MyValidatorFactory.class ) instanceof MyValidatorFactory );
+		assertThat( defaultValidatorFactory ).isNotNull();
```

Comment:
Cool; I think we still can use the simpler form (`org.testng.Assert.assertNotNull()`) in general, i.e. no need to use the fluent form everywhere. Traditionally we tended to use the simple one for not-null and equals and the fluent ones for more complex or type-specific assertions. But I don't mind either way, just meaning to say that from my perspective there is no need to change it wherever we see it.